Documentation/DeepLinking/Index.rst

```diff
@@ -0,0 +1,67 @@
+.. include:: /Includes.rst.txt
+
+
+.. _deeplinking:
+
+============
+Deep Linking
+============
+
+Deep linking allows users to share links to specific pages, content elements and modules
+from within TYPO3's backend.
+
+For example, a user could be writing content for a new page that is about
+to be published. Using deep links, they can share a link to the page
+they are working on or even share a link to the actual content element that contains
+their work and send it to another member of the team for review.
+
+.. _deeplinking-sharing:
+
+Sharing links
+=============
+
+To share a link, look for the Share icon located at the top right hand side of any
+page in the backend.
+
+.. figure:: ../Images/ManualScreenshots/DeepLinking/DeepLinkingIcon.png
+  :alt: The Share icon
+  :class: with-border
+
+
+Linking to a page
+-----------------
+
+The example below would create a deep link to the `About Us` page:
+
+.. figure:: ../Images/ManualScreenshots/DeepLinking/DeepLinkingPageLink.png
+  :alt: Creating a share link in the page module
+  :class: with-border
+
+Linking to a content element
+----------------------------
+
+This example would a create a deep link to the `Introduction` content element, found
+on the `About Us` page:
+
+.. figure:: ../Images/ManualScreenshots/DeepLinking/DeepLinkingContentElementLink.png
+  :alt: Creating a share link to a content element
+  :class: with-border
+
+Linking to backend modules
+--------------------------
+
+It is possible to create links to places within Modules. This example would crea
+
+.. figure:: ../Images/ManualScreenshots/DeepLinking/DeepLinkingSiteModuleLink.png
+  :alt: Creating a link to the backend Sites module
+  :class: with-border
+
+.. _deeplinking-bestpractices:
+
+URLs and deep links
+===================
+
+When creating links to anything in the backend, it is recommended that the
+`Share` button is used to generate the link. Copying the URL from the browser
```

Comment:
Above, _Share_ is also used but without the backticks. I am not sure what would be preferred here. If we refer to something where the text is also used in the UI (which is not the case here) we can use interpreted text roles guilabel, but that is not the case here. 

```
:guilabel:`Share`
```

see also https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/InlineCode.html

I am unsure of the convention here, but I would change it so it is the same in all cases.

(I just saw so much (perceived) overuse of inline markup that I now tend to not use it ... but that is a question of general conventions).

Reply:
Resolved.